src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,13 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+
+    @Override
+    public List<Machine> get() {
```

Comment:
The return type should be parameterized with 'Bulldozer' to prevent the creation of a producer that does not use 'Machine' or its subtype. Change the return type to 'List<Bulldozer>'.